src/Parsers/IAST.h

```diff
@@ -209,7 +210,8 @@ class IAST : public std::enable_shared_from_this<IAST>, public TypePromotion<IAS
             bool always_quote_identifiers_ = false,
             IdentifierQuotingStyle identifier_quoting_style_ = IdentifierQuotingStyle::Backticks,
             bool show_secrets_ = true,
-            LiteralEscapingStyle literal_escaping_style_ = LiteralEscapingStyle::Regular)
+            LiteralEscapingStyle literal_escaping_style_ = LiteralEscapingStyle::Regular,
+            bool print_pretty_type_names_ = false)
```

Comment:
This is getting ridiculous - only because we want to initialize `nl_or_ws` we are having this explicit constructor - way better to have all members default initialized in structure and have `nl_or_ws()` function... though maybe not right now...

Reply:
Yes, it was also ridiculous to me. I thought - I can stand it for this time, but we must change it.